GameRoom.ts

```diff
@@ -12,17 +12,15 @@ import {
   WsData,
 } from './model/WsData';
 
-import { Player } from './model/state/Player';
-import { PhysicsObjectState } from './model/state/PhysicsState';
-import { ItemManager } from './model/ItemManager';
-import { Link } from './model/state/Link';
-import { VoteEntry, VoteStage } from './model/state/VoteState';
+import {Player} from './model/state/Player';
+import {PhysicsObjectState} from './model/state/PhysicsState';
+import {ItemManager} from './model/ItemManager';
+import {Link} from './model/state/Link';
+import {VoteEntry, VoteStage} from './model/state/VoteState';
```

Comment:
Ebenfalls fehlende Spaces

Reply:
Jap wird nicht geforced. Müssen wr mal anpassen

Reply:
Hab nen task aufgemacht dafür https://trello.com/c/dH64khLH/237-backend-linting-doesnt-seem-to-enforce-spaces-for-imports-in-webstorm